app/models/course.rb

```diff
@@ -332,6 +333,18 @@ def rollable?
     is_published? || rollable_withdrawn?
   end
 
+  def rollover_conditions
```

Comment:
I'm wondering if `rollover_conditions` and `show_rollover_button` could be merged into something like (might need to test it):

```suggestion
  def manually_rollable?
    satisfies_rollover = !rolled_over? && recruitment_cycle.current? && recruitment.cycle.next
    satisfies_rollover && %i[empty draft rolled_over].include?(content_status)
  end
```

Either way, not sure if we should drive the method's name from the UI eg (button) it should describe the condition itself i think

Reply:
Done